config/initializers/assets.rb

```diff
@@ -1,13 +1,10 @@
 # Be sure to restart your server when you modify this file.
 
 # Version of your assets, change this if you want to expire all your assets.
-Rails.application.config.assets.enabled = false
```

Comment:
Dropping this line is probably why sprockets has re-appeared

Reply:
That's not quite correct. config.assets isn't a configuration section that is available without sprockets. This is disabled by the removal of sprockets from rails 7. I forgot you removed this and these are managed by webpacker